src/librustdoc/html/static/js/search.js

```diff
@@ -227,6 +634,12 @@ window.initSearch = function(rawSearchIndex) {
         }
 
         function sortResults(results, isType) {
+            var nameSplit = null;
+            if (parsedQuery.elems.length === 1) {
+                var hasPath = typeof parsedQuery.elems[0].path === "undefined";
+                nameSplit = hasPath ? null : parsedQuery.elems[0].path;
+            }
+            var query = parsedQuery.userQuery;
```

Comment:
Since we're now distinguishing nicely between a `parsedQuery` and the `userQuery` that it came from, we should also name this local variable `userQuery`:

```suggestion
            var userQuery = parsedQuery.userQuery;
```

Reply:
:+1: